README.md

```diff
@@ -172,8 +172,8 @@ ec91228cb105db315553499c81918258f52cee9636ea2a4821bdb8226872f54b
 and execute the ORY Hydra command from there:
 
 ```
-$ docker exec -i -t <hydra-container-id> /bin/bash
-# e.g. docker exec -i -t ec91228 /bin/bash
+$ docker exec -i -t <hydra-container-id> /bin/sh
+# e.g. docker exec -i -t ec91228 /bin/sh
```

Comment:
Hi,

Using the docker image: `oryd/hydra:v1.4.10`

this does not seem to work...

```
docker container exec -it <container name> /bin/sh
```

The result is this:

```
exec failed: container_linux.go:349: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory": unknown
```

Reply:
This is from 2017...

Reply:
yup. i can see it's old. but this is still in the readme on docker hub. https://hub.docker.com/r/oryd/hydra/

Search:

```docker exec -i -t <hydra-container-id> /bin/sh```

Reply:
Good point, looks like this isn't properly synced to docker hub any more! https://github.com/ory/hydra/issues/1901